crates/weaver_resolved_schema/src/registry.rs

```diff
@@ -131,6 +131,10 @@ pub struct Group {
     /// This fields is only used for event groups.
     #[serde(skip_serializing_if = "Option::is_none")]
     pub body: Option<AnyValueSpec>,
+    /// Annotations for the group.
+    #[serde(default)]
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub annotations: Option<HashMap<String, YamlValue>>,
```

Comment:
could you please update https://github.com/open-telemetry/weaver/blob/main/schemas/semconv.schema.json and the syntax.md too?

Reply:
Done